### ISSUE TITLE
Lines Added/Lines Removed is empty

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#2487]
<img width="774" alt="Screen Shot 2021-11-17 at 8 08 25 PM" src="https://user-images.githubusercontent.com/427398/142332522-48438851-033a-4175-96cc-7440690f0518.png">

```
TypeError: data[(data.length - 1)] is undefined
    getGitHubData https://dev.api.telescope.cdot.systems/v1/status/js/github-stats.js:27
```

**Comment by HyperTHD** [comment]
I'm not seeing this on my screen, might want to double check this

This is on Chrome, OS is Windows 10 Home
![image](https://user-images.githubusercontent.com/16841702/142337436-4e17b1f4-e71e-472f-a18c-1ff55b38752f.png)

**Comment by humphd** [comment]
I wonder if it was cache, a failed response from GitHub, rate limiting, ...  Today it's working for me.

I'll leave open for now and close if it doesn't happen again.

**Comment by Andrewnt219** [comment]
Probably fixed in #2570
